src/js/lib/content/src/index.js

```diff
@@ -45,10 +45,11 @@ function wasm_instantiate_streaming(resource,imports) {
 
 /// Downloads the WASM binary and its dependencies. Displays loading progress bar unless provided
 /// with `{no_loader:true}` option.
-async function download_content(urlCfg) {
-    let wasm_glue_fetch = await fetch('/assets/wasm_imports.js')
-    let wasm_fetch      = await fetch('/assets/ide.wasm')
-    let loader          = new loader_module.Loader([wasm_glue_fetch,wasm_fetch],urlCfg)
+async function download_content(config) {
+    let wasm_glue_fetch = await fetch(config.wasm_glue_url)
+    let wasm_fetch      = await fetch(config.wasm_url)
+    let loader =
+        new loader_module.Loader([wasm_glue_fetch,wasm_fetch], { no_loader: !config.use_loader })
```

Comment:
instead, use this, please `new loader_module.Loader([wasm_glue_fetch,wasm_fetch],config)` and change the code in `Loader` to check `use_loader` instead of `no_loader`

Reply:
fixed